tutor/src/components/performance-forecast/info-link.cjsx

```diff
@@ -37,5 +37,5 @@ module.exports = React.createClass
     </BS.Tooltip>
 
     <BS.OverlayTrigger placement='right' overlay={tooltip}>
-      <span className='info-link'></span>
+      <button className='info-link'></button>
     </BS.OverlayTrigger>
```

Comment:
Hm, looking at this code, I think it pre-dates the `Icon` component which has built-in tooltip capability.  Since you converted it to use a `<button>` internally, could we swap all the entire render here with something like:

`<Icon type='info-circle' tooltip={MESSAGES[@props.type]} />`

If that works we could also shed all (or most) of the custom `info-link` styles.